game/static/game/css/theme.css

```
@@ -195,3 +195,321 @@
+    position: relative;
+    text-decoration: none;
+    color: var(--text-primary);
+    font-size: 0.95rem;
+    font-weight: 500;
+    transition: color 0.3s ease;
+    padding-bottom: 6px;
+}
+
+.nav-links a::after {
+    content: "";
+    position: absolute;
```

Stylelint (17.14.0)

[error] Duplicate property "position" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)